pkg/ipamd/ipamd.go

```diff
@@ -1181,12 +1175,12 @@ func getWarmPrefixTarget() int {
 	return defaultWarmPrefixTarget
 }
 
-func logPoolStats(total int, used int, cooldownIPs int, maxAddrsPerENI int, Ipv4PrefixDelegation bool) {
-	if !Ipv4PrefixDelegation {
-		log.Debugf("IP pool stats: total = %d, used = %d, IPs in Cooldown = %d, c.maxIPsPerENI = %d", total, used, cooldownIPs, maxAddrsPerENI)
-	} else {
-		log.Debugf("Prefix pool stats: total = %d, used = %d, IPs in Cooldown = %d, c.maxIPsPerENI = %d", total, used, cooldownIPs, maxAddrsPerENI)
+func (c *IPAMContext) logPoolStats(dataStoreStats datastore.DataStoreStats) {
```

Comment:
Could you add a comment for this function ?

Reply:
done